keywords/LiteServNetMono.py

```diff
@@ -26,7 +26,11 @@ def download(self):
         """
 
         # Skip download if packages is already downloaded
-        expected_binary = "{}/couchbase-lite-net-mono-{}-liteserv/net45/LiteServ.exe".format(BINARY_DIR, self.version_build)
+        if self.version_build.startswith("1.2") or self.version_build.startswith("1.3") or self.version_build.startswith("1.4.0"):
```

Comment:
That seems a bit confusing.

What about extracting a method like:

```
# Check if this version has net45
def hasDotNet4Dot5():
    version_prefixes = [
         "1.2", 
         "1.3", 
         "1.4.0",  # For 1.4, the path is net45/LiteServ.exe, for 1.4.0, there is no net45
         ...
]
   .. 
   return True or False
   
```

and then maybe call that function in the various places rather than duplicating the conditional logic